scripts/validation/validate_all_environments.py

```
@@ -0,0 +1,839 @@
+                    )
+                )
+            else:
+                report.results.append(
+                    ValidationResult(
+                        passed=True,
+                        message=f"{url_name} format is valid",
+                        severity="info",
+                        category="urls",
+                    )
+                )
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)